bitbots_body_behavior/src/bitbots_body_behavior/decisions/ball_kick_area.py

```diff
@@ -8,14 +8,10 @@ class BallKickArea(AbstractDecisionElement):
     def __init__(self, blackboard, dsd, parameters=None):
```

Comment:
I am not sure about renaming the Decision completely. Something like BallKickable. As there are not many areas to choose from anymore.

Reply:
Well, I think it can still be interpreted as `BallKickArea` because it defines the area where the ball is kickable, doesn't it?

Reply:
Alright then.